about/changelog.md

```diff
@@ -8,6 +8,74 @@ keywords: [changelog, upgrades, updates, releases]
+
+TimescaleDB v2.17 significantly improves the performance of continuous aggregate refreshes, and contains performance improvements for analytical queries and delete operations over compressed hypertables.
+
+We recommend upgrading at the next available opportunity.
+
+**Highlighted features in TimescaleDB v2.17**
+
+*   Significant performance improvements for continuous aggregate policies:
+
+*   Continuous aggregate refresh is now using `merge` instead of deleting old materialized data and re-inserting.
+
+*   This update can decrease dramatically the amount of data that must be written on the continuous aggregate in the presence of a small number of changes, reduce the `i/o` cost of refreshing a continuous aggregate, and generate fewer Write-Ahead Logs (`WAL`). Overall, continuous aggregate policies will be more lightweight, use less system resources, and complete faster.
+
+*   Increased performance for real-time analytical queries over compressed hypertables:
+
+*   We are excited to introduce additional Single Instruction, Multiple Data (`SIMD`) vectorization optimization to our engine by supporting vectorized execution for queries that group by using the `segment_by` column(s) and aggregate using the basic aggregate functions (`sum`, `count`, `avg`, `min`, `max`).
+
+*   Stay tuned for more to come in follow-up releases! Support for grouping on additional columns, filtered aggregation, vectorized expressions, and `time_bucket` is coming soon.
+
+*   Improved performance of deletes on compressed hypertables when a large amount of data is affected.
+
+*   This improvement speeds up operations that delete whole segments by skipping the decompression step. It is enabled for all deletes that filter by the `segment_by` column(s).
+
+### Enhanced HIPAA Compliance Support in TimescaleDB
+
+TimescaleDB introduces essential updates to facilitate HIPAA (Health Insurance Portability and Accountability Act) compliance, empowering organizations to better protect sensitive healthcare information while simplifying regulatory adherence. We recommend our customers to upgrade to the Enterprise plan to leverage these improvements in data security and auditing:
+
+*   **Introduced Enhanced Encryption and Access Controls**: Timescale encrypts all data at rest and in transit, while providing SAML, SSO, and MFA options for added access security.
+*   **Expanded Audit Logging for HIPAA Compliance**: With the pgAudit and pgSodium extensions, users gain comprehensive logging for key data interactions, capturing user identity, access times, types of operations, and specific data accessed. These logs can be used to identify and investigate potential security incidents.
+*   **Established Breach Notification Protocol**: In case of a PHI data breach, Timescale notifies affected customers within 72 hours, ensuring transparency and compliance with HIPAA requirements.
```

Comment:
```suggestion
*   **Established breach notification protocol**: in case of a PHI data breach, Timescale notifies affected customers within 72 hours, ensuring transparency and compliance with HIPAA requirements.
```